README.md

```diff
@@ -17,85 +17,47 @@ Feel free to visit our more mature SDKs:
 
   
   
+----
 
 
-----
+### What is this project for?
+This project comes to replace and modernize our existing JS offering, and it's a part of a larger effort in Cloudinary. 
```

Comment:
This project comes to replace and modernize our existing JS offering, and it's part of a larger effort in Cloudinary.